src/test/java/org/snomed/snowstorm/core/data/services/RefsetDescriptorUpdaterServiceTest.java

```diff
@@ -89,7 +89,7 @@ void testUpdatingRefSetConcept() throws ServiceException {
 
 		// then
 		List<ReferenceSetMember> members = memberService.findMembers("MAIN", buildMemberSearchRequest(true, Concepts.REFSET_DESCRIPTOR_REFSET), PAGE_REQUEST).getContent();
-		ReferenceSetMember referenceSetMember = members.get(2); // newRefSet2 is 3rd document.
+		ReferenceSetMember referenceSetMember = getReferenceSetMember(members, newRefSet2.getConceptId());
```

Comment:
This is not related to MAINT-1790, however, it is a temperamental test that occasionally failed. I've fixed it by explicitly asking for the member I want.